### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by awvwgk**
Two issue templates for stdlib:

- bug report
- feature proposal

### CODE REVIEW

**Review by jvdp1 — APPROVED** [review]
Useful additions. Thanks

File: .github/ISSUE_TEMPLATE/01_bug.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Something is not working
+labels: bug
+---
+
+<!-- Thanks for taking the time to report a bug -->
+<!-- Fill in all entries in this template to make the process as efficent as possible for everyone involved -->
```
Comment (LKedward) [inline .github/ISSUE_TEMPLATE/01_bug.md:8]:
```suggestion
<!-- Fill in all entries in this template to make the process as efficient as possible for everyone involved -->
```

**Review by LKedward — APPROVED** [review]
Cheers 👍

**Review by milancurcic — APPROVED** [review]
Thanks, looks good.

Is the text in html comment blocks meant to show in the issue template? I don't think it will, but maybe GitHub does something special about it.

**Comment by awvwgk** [comment]
The HTML comments will show up for the author when creating the issue, but will not be present in the finally submitted issue, which is therefore free from these instructions.

**Comment by awvwgk** [comment]
Conda-forge is using those in their issue tempates as well, checkout https://github.com/conda-forge/xtb-feedstock/issues/new

**Comment by milancurcic** [comment]
Perfect, thanks, that didn't occur to me, but now that you explained it it's kinda obvious.

**Review by ivan-pi — APPROVED** [review]
LGTM

**Pull request merged**